packages/server/graphql/types/User.ts

```diff
@@ -442,37 +443,38 @@ const User = new GraphQLObjectType<any, GQLContext>({
         if (viewerMeetingId !== meetingId) {
           return standardError(new Error('Not on team'), {userId})
         }
-        let currentThreshold: number | null = AUTO_GROUPING_THRESHOLD
-        const similarGroupIds = new Set<string>()
         const reflectionsCount = reflections.length
-        const maxGroupSize = Math.min(reflectionsCount, 10)
-        while (currentThreshold) {
+        const maxReductionPercent = 1
+        const maxResultGroupSize = 10
```

Comment:
Sorry I didn't make it clear... I was proposing to put those in constants like `AUTO_GROUPING_THRESHOLD`